data/opt/src/script/build.sh

```diff
@@ -7,6 +7,34 @@ DIR_SRC_PGMODELER=${DIR_SRC}/pgmodeler
 PATH=/opt/mxe/usr/bin:${PATH}
 TOOLCHAIN=x86_64-w64-mingw32.shared
 
+function clone_source() {
+     cd ${DIR_SRC}
+
+     git clone https://github.com/pgmodeler/pgmodeler.git
+}
+
+function clone_plugin_source() {
+     cd ${DIR_SRC}/pgmodeler
+
+     git clone https://github.com/pgmodeler/plugins.git
+
+     cd plugins
+     git checkout develop
+     rmm -rf dummy
+}
+
+function check_version() {
+     local tags_file=$(mktemp)
+
+     cd ${DIR_SRC_PGMODELER}
+
+     git tag >${tags_file}
+
+     if [[ "${1}" =~ $(echo ^\($(paste -sd'|' ${tags_file})\)$) ]]; then
```

Comment:
Would like to see the checks for missing/invalid version reinserted here (see lines 78-94).

Reply:
IMHO we don't need to check for missing tag. We can build for latest source in case tag is missing. I also think if user does not provide tag/version we can get latest tag from the repo itself. 
If user provides a tag it will be used.